### PR TITLE
fix organize data issue for ogb_lsc mag240m

### PR DESCRIPTION
**Pull request by cfangplus**

### CODE REVIEW

**Comment by cfangplus** [comment]
@googlebot I signed it!

File: ogb_lsc/mag/split_and_save_indices.py [hunk 1]
```diff
@@ -46,5 +46,5 @@ def main(argv) -> None:
 
 
 if __name__ == '__main__':
-  flags.mark_flag_as_required('root')
+  # flags.mark_flag_as_required('root')
```
Comment (alvarosg) [inline ogb_lsc/mag/split_and_save_indices.py:49]: Could we keep this as `flags.mark_flag_as_required('data_root')`

File: ogb_lsc/mag/organize_data.sh [hunk 2]
```diff
@@ -46,22 +46,23 @@ if [[ -z "${TASK_ROOT}" ]]; then
...
    "${DATA_ROOT}"/raw/author_writes_paper_edges.npy
-mv "${ROOT}"/mag240m_kddcup2021/processed/paper___cites___paper/edge_index.npy \
+mv "${INPUT_DIR}"/mag240m_kddcup2021/processed/paper___cites___paper/edge_index.npy \
    "${DATA_ROOT}"/raw/paper_cites_paper_edges.npy
+mv "${INPUT_DIR}"/mag240m_kddcup2021/split_dict.pt \
+   "${DATA_ROOT}"/split_dict.pt
 
 # Split and save the train/valid/test indices to the raw directory, with names
-"train_idx.npy", "valid_idx.npy", "test_idx.npy":
+#"train_idx.npy", "valid_idx.npy", "test_idx.npy".
```
Comment (alvarosg) [inline ogb_lsc/mag/organize_data.sh:67]: (add space after "#")

File: ogb_lsc/mag/organize_data.sh [hunk 3]
```diff
@@ -46,22 +46,23 @@ if [[ -z "${TASK_ROOT}" ]]; then
   exit 1
 fi
 
-DATA_ROOT="${TASK_ROOT}"/data
-
+DATA_ROOT="${TASK_ROOT}/data/mag240m_kddcup2021"
```
Comment (alvarosg) [inline ogb_lsc/mag/organize_data.sh:49]: I think if we want to match the structure of the data that is currently store in the bucket, then this additional "/mag240m_kddcup2021"suffix is not necessary?

E.g. directories are something like ".../mag/data/raw/..."

**Review by alvarosg — CHANGES_REQUESTED** [review]
Thanks for the PR!

File: ogb_lsc/mag/split_and_save_indices.py (hunk 1 above)
Comment (cfangplus) [reply ogb_lsc/mag/split_and_save_indices.py:49]: OK

File: ogb_lsc/mag/organize_data.sh (hunk 2 above)
Comment (cfangplus) [reply ogb_lsc/mag/organize_data.sh:67]: OK

File: ogb_lsc/mag/organize_data.sh (hunk 3 above)
Comment (cfangplus) [reply ogb_lsc/mag/organize_data.sh:49]: So, we should also modify one relevant part, which is located in README.md. We could change the "Once this completes, a new directory ROOT/mag240m_kddcup2021/raw will be created" to be "Once this completes, a new directory ROOT/data/raw will be created", right?

Comment (alvarosg) [reply ogb_lsc/mag/organize_data.sh:49]: Yes, you are right, and perhaps also:

we can change:
`/bin/bash organize_data.sh -i ROOT -o ROOT`
to
`/bin/bash organize_data.sh -i DOWNLOAD_DIR -o ROOT`

As I expect that should still work, right?

**Pull request closed without merging**